student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end
```

Comment:
- [x] You did a great job working on this milestone! 💯 However, there's one catch in your code that will cause bugs when creating a Student. If you look closely at your Person class (super) initialize method, it expects two hash parameters: `parent_permission` and `name`. However, in your Student class, you're passing them as regular parameters, causing an argument error. Fixing this is very straightforward:

```suggestion
  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
    super(age, parent_permission: parent_permission, name: name)
    @classroom = classroom
  end
```

You have to pass them as hashes! and that's it. 🤓 🔧